python/llm/example/CPU/HF-Transformers-AutoModels/Model/minicpm/README.md

```diff
@@ -0,0 +1,79 @@
+# minicpm
```

Comment:
The template used here is kind of out of date. Let's refer to the one for Llama 3: https://github.com/intel-analytics/ipex-llm/blob/main/python/llm/example/CPU/HF-Transformers-AutoModels/Model/llama3/README.md